src/devices/Hmc5883l/Hmc5883l.cs

```diff
@@ -58,7 +76,7 @@ public Hmc5883l(I2cDevice sensor, Gain gain = Gain.Gain1090, MeasuringMode measu
         private void Initialize()
         {
             // Details in Datasheet P12
-            byte configA = (byte)(0x70 + _outputRate << 2);
+            byte configA = (byte)(_samplesAmount | _outputRate << 2 | _measurementConfig);
```

Comment:
Looks like by setting _samplesAmount to One as default won't match the existing implementation, so presumably we might want to default to a different value? why was the samples amount before `0x70`, and why don't we have that value in your SamplesAmount enum?

Reply:
Could you please find the description for configuring register A on page 12 of datasheet?
I assume that the existing implementation is not working right.
(0x70 + _outputRate << 2) = 11010000 where _outputRate = 0x04 is default value.
According of datasheet the CRA7 bit must be 0. Also CRA6 and CRA5 bits have values 1 and 0 so it is not default value for "number of samples" configuration. So the 0x70 value is not correct for configuration.

In my implementation we have separated parts for each of configuration bits which are summed by "OR" logical operations.
When we set default values for all configs:
_samplesAmount = 0b_0000_0000(One)
_outputRate << 2 = 0b_0001_0000(Rate15)
_measurementConfig = 0b_0000_0000(Normal)

we have got result: 16 (0b_0001_0000) that is correct value for configuration of register A